src/get-new-version.ts

```diff
@@ -104,6 +121,7 @@ async function promptForNewVersion(operation: Operation): Promise<Operation> {
         { value: 'major', title: `${'major'.padStart(PADDING, ' ')} ${c.bold(next.major)}` },
         { value: 'minor', title: `${'minor'.padStart(PADDING, ' ')} ${c.bold(next.minor)}` },
         { value: 'patch', title: `${'patch'.padStart(PADDING, ' ')} ${c.bold(next.patch)}` },
+        { value: 'semver', title: `${'semver'.padStart(PADDING, ' ')} ${c.bold(next.semver)}` },
```

Comment:
I wonder if we should call it `conventional`, as `semver` is actually a different since than conventional commits